src/core/components/info.jsx

```diff
@@ -66,7 +66,7 @@ class License extends React.Component {
     let url = license.get("url")
 
     return (
-      <div>
+      <div className="info__licence">
```

Comment:
OpenAPI itself spells it `license` (see https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#licenseObject), so let's use that spelling for consistency.

Reply:
.oO(damn Americans…) ok, will change